modules/core/src/v2/keychains.ts

```diff
@@ -120,7 +128,7 @@ Keychains.prototype.updatePassword = function(params, callback) {
  * @param callback
  * @returns {object}
  */
-Keychains.prototype.updateSingleKeychainPassword = function({ keychain, oldPassword, newPassword }) {
+Keychains.prototype.updateSingleKeychainPassword = function({ keychain, oldPassword, newPassword }): any {
```

Comment:
I wasn't sure what to do here with the params, because they are actually in one object and things I tried weren't working